content/040_observability_monitoring_lab/40_scenario.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Lab 1 Scenario"
+chapter: true
+weight: 7
+---
+
+## Lab 1: Find the Root Cause
+
+Here’s the scenario you are trying to troubleshoot and isolate the root cause. 
+
+TravelLogic is an app that facilitates travellers booking their flights online and provides the allied services including hotel booking, sight-seeing, and arranging visits to the well-known places at the destination. 
+
+You, a DevOps specialist, are supporting a TravelLogic app on AWS that uses following AWS services:
+
+- ELB (Application Load Balancer)
+- Host/EC2
```

Comment:
Expand EC2 ( Amazon Elastic Compute Cloud). Required for the first reference, further mentions of the service can be abbreviated.